### DISCUSSION TITLE
Duplicate group names in multiselect options

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#6027] [Q&A]
In the below piece of code, sometimes I get duplicate data which is a legitimate use-case with respect to the feature. but when the group.name is duplicate, I get the following error

```
const getOptions = useCallback(() => {
    return teams.map((team) => ({
      group: team.name,
      items: team.subprojects.map((subproject) => ({
        value: subproject.id,
        label: subproject.name,
      })),
    }));
  }, [teams]);

  return <MultiSelect {...rest} value={boardIds} data={getOptions()} onChange={onBoardSelected} />;
```

![Screenshot 2024-04-02 at 8 44 45 PM](https://github.com/mantinedev/mantine/assets/85650351/55c21e31-a357-4e80-af71-217ff171afe7)

Is there a workaround for such uses cases, I have something like 

```
return teams.map((team, index) => ({
      group: `${team.name}-${index},
```

which works fine, but still wanted to know if there are any other better options that I am missing out

**Comment by startthecode** [comment]
ask here - https://discord.com/channels/854810300876062770/1006447791498870784/threads/1224952126430511164

**Comment by rtivital** [comment]
Duplicate groups are not allowed, it is not planned to bt supported
